Cargo.toml

```diff
@@ -17,8 +17,9 @@ travis-ci = { repository = "fdehau/tui-rs" }
 appveyor = { repository = "fdehau/tui-rs" }
 
 [features]
-default = ["termion"]
+default = ["crossterm"]
```

Comment:
can be changed when finished

Reply:
Yeah I would rather not change the default backend for now. `termion` has been stable for a longer period of time. I could reconsider this in the future when `crossterm` reaches a stable release.

Reply:
agreed